backend/strawberry_forms/converter.py

```diff
@@ -33,15 +35,15 @@ def type_or_optional_wrapped(type_, required):
 def convert_form_field_to_string(field):
     return (
         type_or_optional_wrapped(str, field.required),
-        strawberry.field(description=field.help_text, is_input=True),
+        dataclasses.field(default=field.initial),
```

Comment:
we lose the `help_text` support here because we need to set the default value coming from the form, but as I plan on removing the forms integration anyway I think that's ok